README.md

```diff
@@ -16,16 +16,17 @@ It is also an application generator for building an application based on yaktor.
 
 ## Usage
 
-Typically, you'll want to
+Typically, you'll want to do something like
 
 ```
-$ docker run -it -v "$PWD":/app --rm yaktor/yaktor yaktor create myapp
+$ mkdir myapp && cd "$_"
```

Comment:
should be:
```
$ mkdir myapp
$ cd myapp
$ docker run -it -v "$PWD":/app --rm --entrypoint bash yaktor/base:0.34.1 -c \
 'npm install yaktor@latest'

```
> where post install will `init` (aka migrate) and init will `npm init` if necessary and change to `--safe` instead of `--force`